.github/workflows/test.yml

```diff
@@ -0,0 +1,146 @@
+# This is a GitHub workflow defining a set of jobs with a set of steps.
+# ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions
+#
+name: Test
+
+on:
+  pull_request:
+    branches: ["**"]
```

Comment:
Isn't this the default?

Reply:
Mainly to achieve clear intent signalling.

Reply:
I'll remove it and add a comment on the `on` to help clarify intent